### PR TITLE
try to include custom `id` and `beforeId` props

### PR DESCRIPTION
**Pull request by iduuck**

### CODE REVIEW

File: src/api/canvas-props.ts [hunk 1]
```diff
@@ -2,8 +2,10 @@ import { RenderProps } from "@react-three/fiber";
 import { PropsWithChildren } from "react";
 
 export interface CanvasProps extends Omit<RenderProps<HTMLCanvasElement>, 'frameloop'>, PropsWithChildren {
+  id?: string;
```
Comment (RodrigoHamuy) [inline src/api/canvas-props.ts:5]: You think in here ([link](https://github.com/RodrigoHamuy/react-three-map/blob/main/src/core/use-canvas.ts#L7)) we can do the following, please?

```ts
export interface useCanvasProps extends Omit<CanvasProps, 'id'|'beforeId'> {
```

File: src/mapbox/canvas.tsx [hunk 2]
```diff
@@ -16,11 +16,12 @@ export const Canvas = memo<CanvasProps>((props) => {
   const { id, onAdd, onRemove, render } = useCanvas({ ...props, fromLngLat });
```
Comment (RodrigoHamuy) [inline src/mapbox/canvas.tsx:16]: Can we do this on line 11? So we don't pass these new properties down.
```ts
export const Canvas = memo<CanvasProps>(({id, beforeId...props}) => {
```

**Review by RodrigoHamuy — CHANGES_REQUESTED** [review]
Thanks for doing this ❤️ . Would it be possible to go through my comments and also add a changeset minor file (`yarn changeset` or follow [GH comment ](https://github.com/RodrigoHamuy/react-three-map/pull/70#issuecomment-1771071932) last link)? Thank you 🙏

**Comment by RodrigoHamuy** [comment]
Thanks for doing this! Added a minor change and will be releasing soon https://github.com/RodrigoHamuy/react-three-map/pull/76

**Comment by RodrigoHamuy** [comment]
Thanks for doing this! Added a minor change and will be releasing soon https://github.com/RodrigoHamuy/react-three-map/pull/76

**Comment by iduuck** [comment]
Thanks for the changes, didn't had the free time to tackle those. Sorry.

**Pull request closed without merging**